docs/content/developers/building-contributing.md

```diff
@@ -51,29 +53,29 @@ make
 
 `ddev version` should show you that you are using the correct webtag, and `ddev start` will show it.
 
-It's easiest to do this using gitpod (see above) because gitpod already has `docker buildx` all set up for you and the built ddev is in the $PATH.
+It’s easiest to do this using Gitpod (see above) because Gitpod already has `docker buildx` all set up for you and the built DDEV binary is in the `$PATH`.
 
 ## Pull Requests and PR Preparation
 
-When preparing your pull request, please use a branch name like "2020_<your_username>_short_description" so that it's easy to track to you.
+When preparing your pull request, please use a branch name like `2022_<your_username>_short_description` so it’s easy to identify you as the author.
 
-## Docker Image changes
+## Docker Image Changes
 
-If you make changes to a docker image (like ddev-webserver), it won't have any effect unless you:
+If you make changes to a Docker image (like `ddev-webserver`), it won’t have any effect unless you:
 
-* Push an image with a specific tag by going to the image directory (like containers/ddev-webserver) by just doing `make container VERSION=<branchname>` in the containers/ddev-webserver directory.
-* Multi-arch images require you to have a buildx builder, so `docker buildx create --name ddev-builder-multi --use`
-* You can't push until you `docker login`.
+* Push an image with a specific tag by navigating to the image directory (like `containers/ddev-webserver`), and running `make container VERSION=<branchname>`.
+* Multi-arch images require you to have a Buildx builder, so `docker buildx create --name ddev-builder-multi --use`.
```

Comment:
```suggestion
* Multi-arch images require you to have a buildx builder, so `docker buildx create --name ddev-builder-multi --use`.
```

Hmm, they don't use caps for buildx anywhere, so not sure it's reasonable to do so here. https://docs.docker.com/engine/reference/commandline/buildx/

Reply:
I don’t remember, but I probably took this as (admittedly sketchy) justification:
![Screen Shot 2022-09-25 at 11 02 29 AM@2x](https://user-images.githubusercontent.com/2488775/192158285-9aff2056-8f91-411c-8ef0-923cc977f232.png)

Reply:
Er, no, probably here: https://github.com/docker/buildx

They seem to use `buildx` in lowercase—often in backticks—describing the command and “Buildx” talking about the product. Still not totally consistent, though.

Reply:
Similar treatment here in marketing material: https://www.docker.com/blog/how-to-rapidly-build-multi-architecture-images-with-buildx/

Reply:
I’m going to double down on “Buildx” being the correct product name given the evidence!